ledger/participant-integration-api/src/main/scala/platform/store/dao/events/SqlFunctions.scala

```diff
@@ -30,6 +38,9 @@ private[events] object SqlFunctions {
 
     def arrayIntersectionValues(arrayColumn: String, parties: Set[Party]): String =
       s"array(select unnest($arrayColumn) intersect select unnest(array[${format(parties)}]))"
+
+    def arrayContainedByWhereClause(arrayColumn: String, parties: Set[Party]): String =
+      s"($arrayColumn <@ array[${format(parties)}]::varchar[])"
```

Comment:
In this case, `arrayColumn` (the set of submitters of the command) must be a subset of `parties`. Would we not rather want to check whether there is just "some" overlap, ie at least one of the submitters is contained in the set of requesting parties? (and therefore not using `<@` but `&&`)

Reply:
The `arrayContainedByWhereClause` method is used for two things: filtering out command IDs, and filtering the command completion stream. In the entire PR, I have used the logic that results are only returned if _every_ `actAs` party is contained in the set of parties sending the ledger API request.

E.g., requesting completions for `(Alice, Bob)` would return completions of commands submitted by `(Alice)`, `(Bob)`, and `(Alice, Bob)`. Requesting completions for `(Alice)` would return completions of commands submitted by `(Alice)`, but not completions of commands submitted by `(Alice, Bob)`.

I don't have a strong opinion on this. Is there an argument for using an intersection check, instead of the current subset check?